streamingvisitors/src/tests/matching_elements_filler/matching_elements_filler_test.cpp

```diff
@@ -398,17 +399,6 @@ TEST_F(MatchingElementsFillerTest, ignore_same_element_operator)
     assert_elements(1, "elem_map", { });
 }
 
-TEST_F(MatchingElementsFillerTest, ignore_nested_field)
```

Comment:
I think you missed the rename of the previous test: ignore_same_element_operator() -> and_not_query_node_ignores_all_but_first_child()